packages/demand-bloc-payment/src/providers/AdyenProvider.ts

```diff
@@ -1,5 +1,6 @@
 import AdyenCheckout from '@adyen/adyen-web'
 import CardElement from '@adyen/adyen-web/dist/types/components/Card'
+import { PaymentAction } from '@adyen/adyen-web/dist/types/types'
```

Comment:
Would shorter version `import { PaymentAction } from '@adyen/adyen-web/dist/types'` do the work?

Reply:
Unfortunately, no